src/app/modals/createwallet/createwallet.component.ts

```diff
@@ -55,6 +55,7 @@ export class CreateWalletComponent {
 
   reset() {
     this._modalsService.enableClose = true;
+    this.state.set('model:fullWidth:enableClose', true);
```

Comment:
can you call it `modal` instead of model

Reply:
:D ohh